src/routes/Signin/Signin.tsx

```diff
@@ -91,7 +91,7 @@ export function Signin() {
 
   return (
     <LoginPage
-      className={'pdf-u-background-color-100 pf-u-text-align-center'}
+      className={'pdf-u-background-color-100 pf-u-text-align-left'}
```

Comment:
I'm not sure if `pf-u-text-align-left` is needed but it was added a few commits ago, so I've left it. When it is removed the text is aligned as well.